### ISSUE TITLE
/ui: a11y menu should have a prop to control its own z-index

### ISSUE THREAD

**Issue by mindrones** [nestauk/svizzle#565]
Currently it uses `z-index: var(--z-1000);` which is forcing us to providing a global css var `--z-1000` from the app [1]: it should be a prop

[1]
```css
:root {
	--z-1000: 1000; /* FIXME svizzle's A11yMenu depends on this */
	--z1000: 1000;
}
```
